client/my-sites/plugins/no-permissions-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
```

Comment:
Nit: whenever possible, I'd suggest that we alphabetize the order of imports. This makes them a little bit more readable and predictable.